### PR TITLE
Added the layers of configuration using static method in program, nee…

### PR DESCRIPTION
**Pull request by joni7777**
…d to move to custom nupkg

### CODE REVIEW

**Comment by joni7777** [comment]
This PR is for adding the feature #2

File: BpSeed.Common/Constants.cs [hunk 1]
```diff
@@ -0,0 +1,7 @@
+namespace BpSeed.Common
+{
+    public class Constants
```
Comment (Meir017) [inline BpSeed.Common/Constants.cs:3]: hmm?

File: Components/Components.csproj [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    <PropertyGroup>
+        <TargetFramework>netcoreapp2.2</TargetFramework>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <ProjectReference Include="..\Bp.RouterAliases\Bp.RouterAliases.csproj" />
+    </ItemGroup>
+
+    <ItemGroup>
+      <Reference Include="Microsoft.AspNetCore.Mvc.Core, Version=2.2.0.0, Culture=neutral, PublicKeyToken=adb9793829ddae60">
```
Comment (Meir017) [inline Components/Components.csproj]: just leave as
```xml
<PackageReference Include="Microsoft.AspNetCore.Mvc.Core" Version="2.2.0" />
```

File: Components/Components.csproj [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+        <HintPath>..\..\..\..\..\usr\local\share\dotnet\sdk\NuGetFallbackFolder\microsoft.aspnetcore.mvc.core\2.2.0\lib\netstandard2.0\Microsoft.AspNetCore.Mvc.Core.dll</HintPath>
+      </Reference>
+    </ItemGroup>
+
+    <ItemGroup>
+      <Folder Include="Sample\AutoMapperProfiles" />
```
Comment (Meir017) [inline Components/Components.csproj]: no need to add folders, creating `.cs` files is enough

File: BpSeed.API/Program.cs [hunk 4]
```diff
@@ -1,17 +1,36 @@
...
         public static IWebHostBuilder CreateWebHostBuilder(string[] args) =>
             WebHost.CreateDefaultBuilder(args)
+                .ConfigureAppConfiguration(ConfigConfiguration)
                 .UseStartup<Startup>();
+        
+        private static void ConfigConfiguration(WebHostBuilderContext hostingContext, IConfigurationBuilder config)
+        {
+            var environmentName = hostingContext.HostingEnvironment.EnvironmentName;
+            config
+                .SetBasePath($"{Directory.GetCurrentDirectory()}/Config")
```
Comment (Meir017) [inline BpSeed.API/Program.cs:28]: not sure we need this, you can just prefix the next lines with `./config/`

**Review by Meir017 — CHANGES_REQUESTED** [review]

File: BpSeed.API/Program.cs (hunk 4 above)
Comment (joni7777) [reply BpSeed.API/Program.cs:28]: Not working without it

**Pull request merged**